packages/cli/generators/model/index.js

```diff
@@ -234,6 +234,9 @@ module.exports = class ModelGenerator extends ArtifactGenerator {
 
       // Convert Type to include '' for template
       val.type = `'${val.type}'`;
+      if (val.arrayType) {
```

Comment:
do you mean to include this change in this PR because this PR is only supposed for model template change?